drivers/include/lcd.h

```diff
@@ -267,6 +282,66 @@ void lcd_pixmap(const lcd_t *dev, uint16_t x1, uint16_t x2, uint16_t y1,
 void lcd_write_cmd(const lcd_t *dev, uint8_t cmd, const uint8_t *data,
                    size_t len);
 
+/**
+ * @brief   Raw write command sequence
+ *
+ * The LCD driver allows to write a sequence of commands to the display
+ * controller defined as a sequence of CLP tuples (command, length, parameter).
+ * This can be useful, for example
+ * - to send an initialization sequence to a controller that is not supported
+ *   by a controller-specific driver (see also @ref lcd_params_t::init_seq
+ *   and @ref lcd_init) or
+ * - to write specific configuration parameters that are not covered by the
+ *   controller-specific driver.
+ *
+ * The format of a CLP tuple is:
+ * - one byte the command index
+ * - one byte the length of the following parameters, i.e. the number of bytes
+ * representing the parameters for the command. If the command has no
+ * parameters, the length is 0.
+ * - n bytes the parameters of the command.
+ *
+ * If @ref LCD_DELAY is used as the command index, the command is not sent
+ * to the display controller, but a delay is inserted, where the length
+ * in the CLP tuple then defines the delay in milliseconds.
+ *
+ * The following example shows the initialization sequence for a display
+ * with a ST7789:
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.c}
+ * static const uint8_t st7789_default_init[] =  {
+ *     LCD_CMD_SWRESET, 0,              // Soft Reset
+ *     LCD_DELAY, 120,                  // Soft Reset needs 120 ms if in Sleep In mode
+ *     LCD_CMD_SLPOUT, 0,               // Sleep Out leave Sleep In state after reset
+ *     LCD_DELAY, 120,                  // Sleep Out needs 120 ms
```

Comment:
While I understand why it's not needed, it still irks me a bit that this command does *not* follow the TLV pattern of the other commands.

Reply:
No problem, I can change it. Not using the value in `LCD_DELAY` saves some bytes of code.

Reply:
Changed.